manufacturing/electrical/wiring/README.md

```diff
@@ -47,6 +49,8 @@ BEFORE purchasing any parts.**
 [a15amzn]:  https://www.amazon.com/gp/product/B06ZZ45G7G
 [a16amzn]:  https://www.amazon.com/gp/product/B079MJ1RYN
 [a17amzn]: https://www.amazon.com/CHENBO-Connector-Housing-Assortment-Terminal/dp/B077X8XV2J
+[a12key]: https://www.digikey.com/en/products/detail/molex/0022013037/26433
```

Comment:
reference links should be `a18` and `a19`, and in table above.

Sorry, I know it's a hassle to maintain these manually. Hopefully we will have an automated tool for generating these in the not-so-distant future...

Reply:
Also, as I just tried making this -- one improvement would be to put 3mm shrink tubing around it, so might as well add this to the BOM and instructions:
https://www.amazon.com/uxcell-Polyolefin-Insulation-Shrink-Tubing/dp/B008DFW8JA

Reply:
Can you show me where you applied the shrink tubing?  These types of contacts are not designed to be used with shrink tubing and it is usually not recommended.

Reply:
Can you show me where you applied the heat shrink?  These contacts are not designed to be used with heat shrink and it is generally not recommended.

Reply:
reference links have been updated.